modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/ValidationDate.js

```diff
@@ -215,8 +214,13 @@ const CustomDates = ({
 									'quantity'
 								);
 							}}
+							onChange={(event) => {
+								const {value: newValue} = event.target;
+
+								setQuantity(newValue);
+							}}
 							type="number"
-							value={Math.abs(quantity)}
+							value={quantity !== '' ? Math.abs(quantity) : ''}
```

Comment:
Affirmative expressions are usually easier to understand than denials ones, so flip the condition to make the code more readable:
```suggestion
							value={quantity === '' ? quantity : Math.abs(quantity)}
```